db/system_distributed_keyspace.cc

```diff
@@ -155,17 +155,20 @@ future<> system_distributed_keyspace::stop() {
     return make_ready_future<>();
 }
 
-static const timeout_config internal_distributed_timeout_config = [] {
+static service::query_state& internal_distributed_query_state() {
```

Comment:
I'd feel happier if there was a new query_state for each internal query, but that's outside the scope here. We're making all internal queries behave as if they were one large query.

Reply:
I'll put it on the follow-up list, since I'd prefer to not complicate this patch, it's already a little messy.